scripts/combine/setupCombine.py

```diff
@@ -588,7 +597,7 @@ def assertSample(name, startsWith=["W", "Z"], excludeMatch=[]):
                         for g in cardTool.procGroups["signal_samples"] for m in cardTool.datagroups.groups[g].members},
                 )
 
-    if args.muRmuFPolVar and not isTheoryAgnosticPolVar:
+    if args.muRmuFPolVar and not isPoiAsNoi:
```

Comment:
Is this change correct? Right now we are always using isPoiAsNoi = True (except maybe for one case with unfolding), so this condition would never be entered